libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -1758,134 +1805,63 @@ void WaveTrack::PasteOne(
    WaveTrack &track, double t0, const WaveTrack &other, const double startTime,
    const double insertDuration)
 {
-    //
-    // Pasting is a bit complicated, because with the existence of multiclip mode,
-    // we must guess the behaviour the user wants.
-    //
-    // Currently, two modes are implemented:
-    //
-    // - If a single clip should be pasted, and it should be pasted inside another
-    //   clip, no NEW clips are generated. The audio is simply inserted.
-    //   This resembles the old (pre-multiclip support) behaviour. However, if
-    //   the clip is pasted outside of any clip, a NEW clip is generated. This is
-    //   the only behaviour which is different to what was done before, but it
-    //   shouldn't confuse users too much.
-    //
-    // - If multiple clips should be pasted, or a single clip that does not fill
-    // the duration of the pasted track, these are always pasted as single
-    // clips, and the current clip is split, when necessary. This may seem
-    // strange at first, but it probably is better than trying to auto-merge
-    // anything. The user can still merge the clips by hand (which should be a
-    // simple command reachable by a hotkey or single mouse click).
-    //
-
-    if (other.GetNumClips() == 0)
-        return;
-
-    //wxPrintf("paste: we have at least one clip\n");
-
-    bool singleClipMode = other.GetNumClips() == 1 &&
-        std::abs(startTime) < track.LongSamplesToTime(1) * 0.5;
```

Comment:
The argument `startTime` is now unused, so remove it

Reply:
... which makes me realize that, in `WaveTrack::PasteWaveTrack` and `ClearAndPasteOne`, the `endTime` argument was wrongly passed as `insertDuration` argument to `PasteOne`. I suppose this wasn't noticed by QA because these two outer methods are called with temporary tracks, all with their first clip at 0.